paper.tex

```diff
@@ -63,13 +63,17 @@ \section{Introduction}
 The term “Research software” is also defined somewhat differently within the community.
 Therefore, to avoid ambiguities, we list the ones we use here:\\
 \textbf{Software:}\\
-Source code, documentation, tests, executables and all other artifacts that are created by humans during the development process that are necessary to understand its purpose.\\
+Source code, documentation, tests, executables
```

Comment:
Please use semantic line breaks: this patch introduces line breaks that not only do not change content and make it hard to see what the actual changes were, they would also result in hard-to-read "source code". The same goes for the second change as well.